django/core/validators.py

```diff
@@ -113,10 +115,13 @@ def __init__(self, schemes=None, **kwargs):
 
     def __call__(self, value):
         value = force_text(value)
-        # Check first if the scheme is valid
-        scheme = value.split('://')[0].lower()
-        if scheme not in self.schemes:
-            raise ValidationError(self.message, code=self.code)
+        # If scheme is of special value __all__, ignore the scheme
+        # checks.
+        if self.schemes != '__all__':
```

Comment:
As Claude suggested on the ticket, instead of skipping validation entirely, some minimal validation like alphanumeric + period could be nice. The documentation should describe the validation.